shopinvader_invoice/tests/test_invoice_service.py

```diff
@@ -167,3 +167,25 @@ def test_get_multi_invoice(self):
         data = result.get("data", [])
         self._check_data_content(data, invoices)
         return
+
```

Comment:
I don't see any test for not being able to fetch invoices that belong to other partners. Correct?

Reply:
You're right, I don't write this kind of test for this service